cmd/up.go

```diff
@@ -165,8 +166,19 @@ List of components that kubeapps up installs:
 			return err
 		}
 		clientset, err := kubernetes.NewForConfig(config)
+		nss := []string{KubeappsNS, KubelessNS, SystemNS}
+		for {
```

Comment:
This should be a Watch rather than a polling loop.

Reply:
👍

Reply:
@anguslees Correct me if I am getting your idea wrong. Implementing a Watch is definitely a right way to go but I am afraid of having to import the _heavy_ kubectl package. See https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/resource/get.go#L444